PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -227,8 +227,15 @@ def miniAOD_customizeCommon(process):
     photon_ids = ['RecoEgamma.PhotonIdentification.Identification.cutBasedPhotonID_Spring15_25ns_V1_cff',
                   'RecoEgamma.PhotonIdentification.Identification.cutBasedPhotonID_Spring15_50ns_V1_cff',
                   'RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_Spring15_25ns_nonTrig_V2p1_cff',
-                  'RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_Spring15_50ns_nonTrig_V2p1_cff']
-    switchOnVIDPhotonIdProducer(process,DataFormat.MiniAOD, task)
+                  'RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_Spring15_50ns_nonTrig_V2p1_cff',
+                  'RecoEgamma.PhotonIdentification.Identification.cutBasedPhotonID_Spring16_V2p2_cff',
+                  'RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_Spring16_nonTrig_V1_cff']
+    switchOnVIDPhotonIdProducer(process,DataFormat.AOD, task) 
```

Comment:
This means that effectively for the isolation calculation we have to use inputs as in AOD case. Of course, we have tried to use packedPFCandidates, but the problem is that at this step reducedEgamma is the input collection, and for example associatedPackedPFCandidates() is not available (at least it was failing). In CITK it is possible to use slimmedPhotons together with packedPFCandidates or gedPhotons (or reducedEgamma in this case) together with pfNoPileUpCandidates.